transitmatters/t-performance-dash
# MBTA speed restrictions data in SZT

This PR pulls in speed restrictions data from the MBTA open data portal, and shows it in the SZT. The URL for the data is hard-coded to February for now, but they've told us they're going to start publishing regular updates as CSVs soon.

The design is pretty basic, but all the data is getting piped in and there's plenty we can do with this. One thought is that we might want to show this data in the "Locations" bar chart view, too.

_Screenshot:_

![image](https://user-images.githubusercontent.com/2208769/229978400-f98625d5-2727-48c2-8f68-81660ed1cc6e.png)

_Test plan:_

- Run `npm run get-sz-data` and verify that it now creates a `public/static/slowzones/speed_restrictions.json`
- Open the SZT and verify that you can see the new data in the map tooltips.
